### ISSUE TITLE
Add map info

### ISSUE THREAD

**Issue by pnxenopoulos** [pnxenopoulos/csgo#40]
Can/should add a dataframe/json containing info on the parsed nav files. Shouldn't be too big and can enable easier analysis and visualization.

**Comment by pnxenopoulos** [comment]
Can use something like this

```
package main

import (
	"flag"
	"fmt"
	"os"

	gonav "github.com/pnxenopoulos/csgonavparse"
	metadata "github.com/markus-wa/demoinfocs-golang/v2/pkg/demoinfocs/metadata"
)

func main() {
	fl := new(flag.FlagSet)

	navPathPtr := fl.String("map", "", "Map name `path`")

	err := fl.Parse(os.Args[1:])
	if err != nil {
		panic(err)
	}

	currentMap := *navPathPtr
	currentMap = currentMap[:len(currentMap)-4]
	mapMetadata := metadata.MapNameToMap[currentMap]

	// Read in parser
	fNav, _ := os.Open(currentMap + ".nav")
	parserNav := gonav.Parser{Reader: fNav}
	mesh, _ := parserNav.Parse()

	var NWX, NWY, SEX, SEY float64 = 0.0, 0.0, 0.0, 0.0
	
	//fmt.Printf("MapName,AreaId,AreaName,NorthWestX,NorthWestY,NorthWestZ,SouthEastX,SouthEastY,SouthEastZ,Connections,HidingSpots,EarliestOccupyTimeFirstTeam,EarliestOccupyTimeSecondTeam \n")
	
	for _, currArea := range mesh.Areas {
		name := "NA"
		if currArea.Place != nil {
			name = currArea.Place.Name
		}
		NWX, NWY = mapMetadata.TranslateScale(float64(currArea.NorthWest.X), float64(currArea.NorthWest.Y))
		SEX, SEY = mapMetadata.TranslateScale(float64(currArea.SouthEast.X), float64(currArea.SouthEast.Y))
		fmt.Printf("%s,%d,%s,%f,%f,%f,%f,%f,%f,%d,%d,%f,%f \n", currentMap, currArea.ID, name, NWX, NWY, currArea.NorthWest.Z, SEX, SEY, currArea.SouthEast.Z, len(currArea.Connections), len(currArea.HidingSpots), currArea.EarliestOccupyTimeFirstTeam, currArea.EarliestOccupyTimeSecondTeam)
	}
}
```

**Comment by pnxenopoulos** [comment]
Added in bcd344054c25c82ad0c88c7c27c5559a5c86d987

**Comment by pnxenopoulos** [comment]
Added tests and references in code in 6371d0f608e0b53c055be3e99d833861e375d656

**Comment by pnxenopoulos** [comment]
```
import pandas as pd
import numpy as np

from tqdm import tqdm

df = pd.read_csv("map_info.csv")

fixed_areas = []

for _, r in tqdm(df[df["AreaName"].isna()].iterrows()):
    map_areas = df[df["MapName"] == r["MapName"]]
    min_dist = 10000000
    closest_area = r
    center_r_x = (r["SouthEastX"] - r["NorthWestX"])/2
    center_r_y = (r["SouthEastY"] - r["NorthWestY"])/2
    center_r_z = (r["SouthEastZ"] - r["NorthWestZ"])/2
    for _, a in map_areas.iterrows():
        center_a_x = (a["SouthEastX"] - a["NorthWestX"])/2
        center_a_y = (a["SouthEastY"] - a["NorthWestY"])/2
        center_a_z = (a["SouthEastZ"] - a["NorthWestZ"])/2
        dist = np.sqrt( (center_r_x - center_a_x)**2 + (center_r_y - center_a_y)**2 + (center_r_z - center_a_z)**2 )
        if (dist < min_dist) & (dist > 0) & (~pd.isna(a["AreaName"])):
            closest_area = a
    fixed_areas.append(closest_area["AreaName"])

df.loc[df.AreaName.isna(),"AreaName"] = fixed_areas

df.to_csv("map_nav.csv", index=False)
```
fixed in a88288c3b3e3a6970563b38a642acbcae17d6820
